project-koku/koku
# remove local differentiation from HCS provider types

## Description

This change will remove the local vs non-local differentiation from providers so we do not have duplicate SQL files to maintain.

## Testing

1. Checkout Branch
2. set `ENABLE_HCS_DEBUG=True` in your .env so you don't have to worry about unleash
3. Restart Koku
4. Load the test customer data
- `make create-test-customer` followed by `make load-test-customer-data`
5. Check minio and ensure that you see HCS reports for AWS/Azure/GCP in the bucket.

## Notes

...
